package.json

```diff
@@ -33,11 +33,11 @@
     },
     "dependencies": {
         "@braze/web-sdk-core": "^3.0.0",
-        "@emotion/core": "^10.0.35",
         "@cypress/skip-test": "^2.5.0",
+        "@emotion/core": "^10.0.35",
         "@guardian/ab-core": "^2.0.0",
         "@guardian/ab-react": "^2.0.1",
-        "@guardian/atoms-rendering": "^2.0.9",
+        "@guardian/atoms-rendering": "^3.0.0",
```

Comment:
Bump to get get passed Webpack not being able to bundle `react-dom/server` which kept erroring with this:

```
 ERROR in ./node_modules/@guardian/atoms-rendering/dist/index.modern.js 10:0-55
  Module not found: Error: Package path ./compat/server is not exported from package /Users/gareth_trufitt/code/dotcom-rendering/node_modules/preact (see exports field in /Users/gareth_trufitt/code/dotcom-rendering/node_modules/preact/package.json)
   @ ./src/web/components/App.tsx 113:9-115:30
   @ ./src/web/components/HydrateApp.tsx 9:0-28 41:38-41
   @ ./src/web/browser/react/init.ts 4:0-57 26:2-12
```

Reply:
I guess this PR negates the problems in @liywjl 's [one here](https://github.com/guardian/dotcom-rendering/pull/2142/files) then, which is nice

Reply:
I used this PR as a base for the Youtube changes. Looks like yes this PR has fixed the `jest` issues I had 🎊 
Why or how? 🤷  Thats beyond me. 

Thanks a million Garth!!!